spec/lib/synchronized_model/message_receive_spec.rb

```diff
@@ -106,6 +106,23 @@ def updated_at=(_value); nil; end
           expect { subject }.to raise_error(RuntimeError)
         end
       end
+
+      context 'it\'s a create' do
+        let(:updated_at_was) { nil }
+        let(:updated_at) { (Time.now + 1) }
+        let(:errors) { [] }
+
+        it 'calls save' do
+          subject
+          expect(mock_model_sequel).to have_received(:save)
+        end
+
+        it 'raises an error if validations fail' do
+          errors << ['error']
+
+          expect { subject }.to raise_error(RuntimeError)
+        end
+      end
```

Comment:
This test doesn't fail if I remove my change. Not sure how to test this. Everything is very mocked because there's no actual Sequel or ActiveRecord gem in here.